docs/install-kafka-operator.md

```diff
@@ -156,44 +228,97 @@ Install the [Prometheus operator](https://github.com/prometheus-operator/prometh
     --set prometheus.enabled=false
     ```
 
-### Install {{< kafka-operator >}} with Helm {#kafka-operator-helm}
+1. Verify that prometheus-operator has been deployed and is in running state.
+
+    ```bash
+    kubectl get pods -n prometheus
+    ```
+
+    Expected output:
 
-You can deploy {{< kafka-operator >}} using a [Helm chart](https://github.com/banzaicloud/koperator/tree/master/charts). Complete the following steps.
+    ```bash
+    NAME                                                   READY   STATUS    RESTARTS   AGE
+    prometheus-kube-prometheus-operator-646d5fd7d5-s72jn   1/1     Running   0          15m
+    ```
+
+### Install {{< kafka-operator >}} with Helm {#install-kafka-operator-with-helm}
+
+You can deploy {{< kafka-operator >}} using its [Helm chart](https://github.com/banzaicloud/koperator/tree/master/charts). Complete the following steps.
 
 1. Install the {{< kafka-operator >}} CustomResourceDefinition resources (adjust the version number to the {{< kafka-operator >}} release you want to install). This is performed in a separate step to allow you to uninstall and reinstall {{< kafka-operator >}} without deleting your installed custom resources.
 
     ```bash
     kubectl create --validate=false -f https://github.com/banzaicloud/koperator/releases/download/v{{< param "versionnumbers-sdm.koperatorCurrentversion" >}}/kafka-operator.crds.yaml
     ```
 
-1. Add the Banzai Cloud repository to Helm.
+    ```
+    customresourcedefinition.apiextensions.k8s.io/cruisecontroloperations.kafka.banzaicloud.io created
+    customresourcedefinition.apiextensions.k8s.io/kafkaclusters.kafka.banzaicloud.io created
+    customresourcedefinition.apiextensions.k8s.io/kafkatopics.kafka.banzaicloud.io created
+    customresourcedefinition.apiextensions.k8s.io/kafkausers.kafka.banzaicloud.io created
+    ```
+
+1. If you are installing {{< kafka-operator >}} on Red Hat OpenShift, you need to elevate the permissions of the namespace you install Koperator and your Kafka resources into.
+
+    ```bash
+    oc adm policy add-scc-to-group anyuid system:serviceaccounts:kafka
```

Comment:
It is not clear to me what is the difference in those two commands: 
oc adm policy add-scc-to-group anyuid system:serviceaccounts:kafka
oc adm policy add-scc-to-group anyuid system:serviceaccounts:{NAMESPACE_FOR_KOPERATOR}

Reply:
The difference is if the user installs Koperator into another namespace, let's say `kafka-user-apps` then the command needs to be `oc adm policy add-scc-to-group anyuid system:serviceaccounts:kafka-user-apps`.